pom.xml

```diff
@@ -90,7 +90,8 @@
         <awaitility.version>4.2.1</awaitility.version>
         <embedded-redis.version>0.9.0</embedded-redis.version>
         <errorprone.version>2.15.0</errorprone.version>
-        <jedis.version>5.1.2</jedis.version>
+        <jedis.version>5.1.0</jedis.version>
```

Comment:
Why did you downgrade Jedis from `5.1.2` to `5.1.0`?

Reply:
Reverted, I had problems when dependencies were added transitively, but I will test it again